website/static/js/pages/login-page.js

```diff
@@ -1,6 +1,25 @@
 /**
  * Login page
  */
-var LogInForm = require('js/signIn');
+var $ = require('jquery');
 
+var LogInForm = require('js/signIn');
 new LogInForm.SignIn('#logInForm');
+
+var SignUp = require('../signUp.js');
+new SignUp('#signUpScope', '/api/v1/register/', $('#campaign').val());
```

Comment:
Agreed, but this code appears shamelessly copy/pasted from https://github.com/CenterForOpenScience/osf.io/blob/6661fdc11aef164c6bce25a34ac9a83a0bb1a16c/website/static/js/pages/home-page.js. If you feel strongly about this I can change it.